internal/db/schema/migrations/oss/postgres/56/02_add_data_key_foreign_key_references.up.sql

```diff
@@ -244,6 +244,7 @@ begin;
     'This view should only be used when issuing credentials from a Vault credential library. Each row may contain encrypted data. '
     'This view should not be used to retrieve data which will be returned external to boundary.';
 
+  -- Replaced in 02_add_ssh_cert_to_vault_cred_library_view.up.sql
```

Comment:
This comment looks like it is in the wrong place. The `credential_vault_token_renewal_revocation` view is not being replaced, I think this comment is meant for the view above?